great_expectations/data_context/types/base.py

```diff
@@ -1776,18 +1784,34 @@ def action_list(self, value: List[dict]):
     def site_names(self):
         return self._site_names
 
+    @site_names.setter
+    def site_names(self, value: Union[list, str]):
+        self._site_names = value
+
     @property
     def slack_webhook(self):
         return self._slack_webhook
 
+    @slack_webhook.setter
+    def slack_webhook(self, value: str):
+        self._slack_webhook = value
+
     @property
     def notify_on(self):
         return self._notify_on
 
+    @notify_on.setter
```

Comment:
Do we really use this setter?..

Reply:
Hm, good catch. I added it because I thought it would be needed when the `update` method was called at runtime (within `.run` method), but after checking code, realized the SimpleCheckpoint-specific keys wouldn't be processed. 

Removed those setters for now and adding asana task to discuss desired runtime behavior of SimpleCheckpoint.